### DISCUSSION TITLE
Open issue and PR to add `easy` to the ignore list of EddieBot

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#586] [First-Timers] [answered]
It would be really great, if someone(first-timers) could open an issue for adding the word `easy` on the [Eddiebot](https://github.com/EddieJaoudeCommunity/EddieBot) repository and maybe also open a PR for resolvin the issue.
🟩 

#### Where to open:
https://github.com/EddieJaoudeCommunity/EddieBot/issues

### Description:
EddieBot keeps on flagging the word "easy" and "easily", which in the server, mostly refers in contexts like:
```
Using github actions on repositories makes your life easier.
```

### Screenshot: 
![Screenshot 2020-12-23 at 6 46 50 PM](https://user-images.githubusercontent.com/62864373/103001286-b1f21800-4552-11eb-8678-c6f91db46290.png)

### How to resolve: 
Add the word `easy` to the ignore list(ALEX):

https://github.com/EddieJaoudeCommunity/EddieBot/blob/52dbcb3df01024a19c1576c0442224c8e21c1c1f/src/config.ts#L87-L110

---
ISSUE opened by: @Utkarsh299-tech 
PR opened by: <>

**Comment by eddiejaoude** [comment]
Thanks @Vyvy-vi 👍

**Comment by Vyvy-vi** [comment, marked as answer]
An issue for this has been opened by @Utkarsh299-tech
[EddieBot#407](https://github.com/EddieJaoudeCommunity/EddieBot/issues/407)

**Reply by Utkarsh299-tech** [reply]
> An issue for this has been opened by @Utkarsh299-tech
> [EddieBot#407](https://github.com/EddieJaoudeCommunity/EddieBot/issues/407)

Yes, I've opened an issue for this.
